### DISCUSSION TITLE
New backup: repository not found

### DISCUSSION THREAD

**Discussion by andreapx** [cupcakearmy/autorestic#369] [General]
I've just installed autorestic and I've never used it nor restic.

I've created this config file:
```
version: 2
locations:
  docker-data:
    from: /var/docker
    to: docker-data-backup
    options:
      forget:
        keep-last: 10
        keep-daily: 7
        keep-weekly: 4
        keep-monthly: 6
        keep-yearly: 1

backends:
  docker-data-backup:
    type: local
    path: /mnt/nas01/docker_data
```

but when I try to run a backup with `sudo autorestic backup -a`

I get: 
```
Fatal: unable to open config file: stat /mnt/nas01/docker_data/config: no such file or directory
Is there a repository at the following location?
/mnt/nas01/docker_data

docker-data@docker-data-backup:
Fatal: unable to open config file: stat /mnt/nas01/docker_data/config: no such file or directory
Is there a repository at the following location?
/mnt/nas01/docker_data
exit status 1
```

I can't find anything about creating a repository in help [here](https://autorestic.vercel.app/backend).
I've tried with `autorestic init -r /mnt/nas01/docker_data` but init is not recognized.

What should I do?

Thanks!

**Comment by Keisir** [comment]
Try running `autorestic check`. This command ensures that all backends (restic repositories) are created
Quickstart: https://autorestic.vercel.app/quick#check
check reference: https://autorestic.vercel.app/cli/check

**Reply by andreapx** [reply]
That was it, thanks!
